### PR TITLE
server: wrap excessively long log message lines (>100 characters)

### PR DESCRIPTION
**Pull request by petermattis**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4640)

### CODE REVIEW

**Comment by tamird** [comment]
LGTM

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[server/context.go, line 151 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4640#-KBNyEvxIRJwMPH7sdyH:-KBNyEvxIRJwMPH7sdyI:-383427462) ([raw file](https://github.com/cockroachdb/cockroach/blob/6bb3a98ed8697c98f7db99a8b7e91378983d2b8e/server/context.go#L151)):*
while you're here, could you use go-humanize to format these values?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4640#-:-KBNyNr_pM6JHuxtS6AP:676834233)*

File: server/context.go [hunk 1]
```diff
@@ -147,7 +147,9 @@ func getDefaultCacheSize() uint64 {
 	mem := sigar.Mem{}
 	if err := mem.Get(); err != nil {
 		if log.V(1) {
-			log.Infof("can't retrieve system memory information (%s), setting default rocksdb cache size to %dMB", err, defaultCacheSize>>20)
+			log.Infof("can't retrieve system memory information (%s)\n"+
+				"\tsetting default rocksdb cache size to %dMB",
```
Comment (tschottdorf) [inline server/context.go]: that won't mess with clear-text  log parsing?

Comment (petermattis) [inline server/context.go]: Nope. The log parsing looks for lines that start with the log message header.

Comment (tschottdorf) [inline server/context.go]: Ok. And it's able to attach the second line to the original message header?

Comment (petermattis) [inline server/context.go]: Yep. We split the log file at log message header boundaries and grab everything from the start of a message header to the beginning of the next as a single message.

**Comment by petermattis** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[server/context.go, line 151 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4640#-KBNyEvxIRJwMPH7sdyH:-KBNyqOaXvJeP-Q4QoPE:1251790129) ([raw file](https://github.com/cockroachdb/cockroach/blob/6bb3a98ed8697c98f7db99a8b7e91378983d2b8e/server/context.go#L151)):*
This is a log message that will likely never be viewed. Doesn't seem worth it in my opinion. Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4640)*

**Comment by tschottdorf** [comment]
LGTM

**Comment by tamird** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4640)*

**Pull request merged**